### DISCUSSION TITLE
Event when conversations are resolved and unresolved?

### DISCUSSION THREAD

**Discussion by warsaw** [community/community#24727] [Apps, API and Webhooks]
<p>It doesn’t appear that any events get triggered when a PR conversation is resolved or unresolved.  Am I missing it?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You are correct. There is currently no webhook or other event for these activities.</p>
<p>Let us know if you have more questions.</p>

**Comment by warsaw** [comment]
<p>How should I make a feature request for this?</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/warsaw">@warsaw</a>,</p>
<p>This board is also an appropriate place for feature requests.  I’ve logged your feature request in our internal feature request list!</p>
<p>Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>

**Comment by fpiecho** [comment]
<p>Hi! I wanted to know if the feature request for this has been addressed</p>

**Comment by chdsbd** [comment]
<p>This hasn’t been addressed, but I would be thrilled if this feature was added.</p>

**Comment by matpen** [comment]
I would find this very useful, too.
